src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -88,9 +88,7 @@ def test_inplace_add(self):
             form={'type_name': u'BrowserAdd__zope.site.folder.Folder'})
         body = response.unicode_normal_body
         self.assertIn('type="hidden" name="type_name"', body)
-        #self.assertIn('input name="new_value"', body)
         self.assertIn('type="submit" name="container_cancel_button"', body)
-        #self.assertIn('type="submit" name="container_rename_button"', body)
```

Comment:
They were failing because they were conditional on certain registrations being done by packages that were no longer present. I hadn't bothered to track them down because the coverage for adding.py didn't change.

But I went ahead and tracked them down just for you 😄 I've added them to the local `ftesting.zcml`.